module/dice/aggregate-damage-rolls.mjs

```diff
@@ -60,14 +60,14 @@ function chunkTerms(terms, type) {
 
   for ( let term of terms ) {
     // Plus or minus operators split chunks
-    if ( (term instanceof OperatorTerm) && ["+", "-"].includes(term.operator) ) {
+    if ( (term instanceof foundry.dice.terms.OperatorTerm) && ["+", "-"].includes(term.operator) ) {
```

Comment:
This could probably also have the 'import at top of file' treatment since this file deals with rolls specifically.